xds/internal/xdsclient/dump_test.go

```diff
@@ -43,6 +44,8 @@ import (
 	"google.golang.org/grpc/xds/internal/xdsclient/bootstrap"
 )
 
+const defaultTestWatchExpiryTimeout = 500 * time.Millisecond
```

Comment:
Delete if unused?

Reply:
This var is defined in the non`_test` package. This is a different package now.

Reply:
Ohh, I see. So it's defined twice right now?  OK, that's fine then.